tpm2/src/getkey/main.c

```diff
@@ -229,11 +275,18 @@ int main(int argc, char **argv)
     BIO_free_all(bio);
     SSL_CTX_free(ctx);
     kmyth_clear_and_free(clientPrivateKey_data, clientPrivateKey_size);
+    free(sdo_orig_fn);
```

Comment:
If we wanted to, I think we could also move the free of the sealed data object filename variable (sdo_orig_fn) up. I think we are done with it after the tpm2_kmyth_unseal() call.